### ISSUE TITLE
Selectors in aggregation view - keyboard navigation doesn't work

### ISSUE THREAD

**Issue by sgauruseu** [enonic/app-contentstudio#4622]
1. Type an user name in "Last modified by" options filter input

**Issue**:  keyboard navigation doesn't work in the selector:

![image](https://user-images.githubusercontent.com/3728712/171348583-05bf6f18-b02c-4ab4-8336-58b2c9d2e37b.png)
